virttest/graphical_console.py

```diff
@@ -0,0 +1,97 @@
+"""
+Graphical Console
+"""
+
+import time
+
+
+def uniform_linear(duration, rate):
+
+    def motion(src, dst):
+        pass
+
+    return motion
+
+
+class DummyConsole(object):
+
+    """
+    Dummy console
+    """
+
+    KEY_DOWN = True
+    KEY_UP = False
+
+    BTN_DOWN = True
+    BTN_UP = False
+
+    SCROLL_FORWARD = 1
+    SCROLL_BACKWARD = -1
+
+    def __init__(self, vm):
+        self._vm = vm
+
+    def key_down(self, key):
+        self._key_event(self.KEY_DOWN, key)
+
+    def key_up(self, key):
+        self._key_event(self.KEY_UP, key)
+
+    def key_press(self, key, hold_time=0):
+        if isinstance(key, str):
+            key = (key)
+        self.key_down(key)
+        if hold_time:
```

Comment:
is it better to have one method like `def hold_time` to do the sleep?

Reply:
updated, thanks~